spec/features/admins/admin_user_sees_user_profile_spec.rb

```diff
@@ -53,12 +53,33 @@
       click_button "Submit"
 
       expect(page).to have_content("User profile has been updated")
-      expect(page).to have_content("Username: editedname")
+      expect(page).to have_content("Name: editedname")
       expect(page).to have_content("Email: new_email@email.com")
 
       expect(current_path).to eq(admin_user_path(user))
     end
 
+    it 'wont allow me to update without all the right info' do
```

Comment:
Thanks for testing this sad path! didn't think of it last night, makes total sense